setup.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# Copyright (c) Juptyer Development Team.
+# Distributed under the terms of the Modified BSD License.
+
+from __future__ import print_function
+
+import os
+import sys
+
+from distutils.core import setup
+
+pjoin = os.path.join
+here = os.path.abspath(os.path.dirname(__file__))
+
+
+setup_args = dict(
+    name                = 'jupyter',
+    version             = '4.0.0',
+    description         = "Jupyter metapackage. Install all the Jupyter components in one go.",
+    long_description    = """Install the Jupyter system, including the notebook, qtconsole, and the IPython kernel.""",
+    author              = "Jupyter Development Team",
+    author_email        = "jupyter@googlegroups.org",
+    py_modules          = ['jupyter'],
+    install_requires    = [
+        'notebook',
+        'qtconsole',
+        'jupyter-console',
+        'nbconvert',
+        'ipykernel',
```

Comment:
Should this also depend on `ipython>=4`, so that people installing it who have older versions of IPython installed get upgraded? Alternatively, we could merge ipython/ipykernel#33 and push out a new ipykernel release before we release this package.